### ISSUE TITLE
The world doesn't load

### ISSUE THREAD

**Issue by PzmuV1517** [JayexDesigns/cc-tweaked-potato-amazon#1]
Hi, I tried loading the world that you provided so I could better understand the programs but when loading it said that some data packes failed and that I could open the world in "safe mode" or quit to the title screed. When clicking on safe mode minecraft stalls for about 5 minutes then does not respond again after 5 minutes minecraft crashes completely! What should I do? 

Recreation steps: 
1. Download zip file
2. Install the mod pack using CurseForge
3. Open the folder of the minecraft instance and unzip the world.zip into the saves folder
4. Run the minecraft instance
5. Open the world

**Comment by PzmuV1517** [comment]
I will try using a beefier PC and make the ram allocation wayyyyyy larger tomorrow.

**Comment by JayexDesigns** [comment]
It's a project I did a while ago and it's not been updated in a while, if you're using the correct version and have the mod installed I don't really know what's failing, in the worst case the code it's still there and can be used in any world

**Comment by PzmuV1517** [comment]
Yeah, I understand that the code can be used in any world but my problem is that I don't understand the turtle placement fully and a 3d model of the place would be better. I will try again with a better pc like I said and then I will comment the results! 

Thanks for the suport!

**Comment by JayexDesigns** [comment]
Right now I don't have a good PC available and I will not have one for a few weeks, then I'll test it again if your other PC doesn't work either, for now I don't have a clue on what could it be

**Comment by PzmuV1517** [comment]
Hi, I tried opening it on a better pc but the world itself may have been corrupted after uploading to github. If you still have the world can you please try sending it through a file shareing service?

**Comment by PzmuV1517** [comment]
Another issue I have is the it gives me an issue when trying to run the harvest.lua file it says that there is an issue at line 81 something about reciving a null instead of a number.

**Comment by JayexDesigns** [comment]
Well now I factory reset my PC multiple times since doing this project, I'm currently in another country from it and honestly I've forgotten how to code in Lua. When I get back I can try to see what went wrong but since it was a project I did for the memes and for a server that no longer exists I don't know if it's worth the effort

**Comment by PzmuV1517** [comment]
1 year older me here... I am trying to reconstruct the world. I saw a few days ago a tutorial on how to restore corrupted worlds. If that doesn't work I'll start making it from the pictures you provided. God speed to myself.

**Comment by PzmuV1517** [comment]
I started remaking it because me and some friends started a server and I am the only one that is not trying to get spells for some stupid book instead I'm building my empire of baked potatoes and on command drone strikes.😆

**Comment by PzmuV1517** [comment]
I managed to restore the world. Im adding some stuff from myself and making multiple versions of the shop to be better in certain situations. Also adding some blocks to easily make schematics of the store.( PS removing the long turtle line that is used as a pipeline improves performance)

**Comment by PzmuV1517** [comment]
Update - replaced the turtle line with hoppers and added custom bill board / logo at the front of the store. Renamed Potato Amazon into Pamazon and made the official logo the text PAZ with a YouTube buddy pointing from the "P" to the "Z".
